### PR TITLE
CrossQ

### PR DESCRIPTION
**Pull request by JasperHoffmann**
## CrossQ algorithm with ZOP variant

> **Note:** This PR depends on #273  and should be merged after it.

Adds the CrossQ off-policy RL algorithm and its ZOP (zeroth-order policy)
variant. Introduces a batchnorm option to the MLP module (required by CrossQ's
critic architecture) and finalises the CrossQ training scripts for HVAC.
### Commits
- feat(mlp): add batchnorm option
- feat(CrossQ): add CrossQ
- feat(CrossQ): add CrossQ for ZOP
- feat(Cross-Q): finalized cross-q scripts and trainer
- refactor(Cross-Q): cleaned crossq.py
### Files changed
- `leap_c/torch/rl/crossq.py`
- `leap_c/torch/rl/crossq_zop.py`
- `leap_c/torch/nn/mlp.py`
- `scripts/hvac/run_crossq.py`
- `scripts/hvac/run_crossq_zop.py`

### CODE REVIEW

File: leap_c/torch/nn/mlp.py [hunk 1]
```diff
@@ -81,11 +81,17 @@ class MlpConfig:
         activation: The activation function to use in the hidden layers.
         weight_init: The weight initialization method to use for the hidden layers. If `None`, no
             initialization will be applied.
+        batchnorm: Whether to use batch normalization between layers.
+
+    Todo (Jasper):
+        Consider switching to RL-specific BatchNorm with stable running average statistics. Current,
```
Comment (FilippoAiraldi) [inline leap_c/torch/nn/mlp.py]: Doesn't `BatchNormXd` already use running statistics instead of batch stats by default (`track_running_stats=True`)?

Comment (JasperHoffmann) [reply leap_c/torch/nn/mlp.py]: True, I was reading the description here:
https://docs.pytorch.org/rl/0.7/_modules/torchrl/modules/models/batchrenorm.html

Thus, this "more stable" version uses minibatch stats in the beginning of the training and then switches to running stats. Initial experiments did on `hvac` lead to worse results using this implementation.

Will update the Todo description.
